server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -153,7 +153,7 @@ export class ScribeLambdaFactory extends EventEmitter implements IPartitionLambd
                 // is okay. Conceptually this is similar to default checkpoint where logOffset is -1. In this case,
                 // the sequence number is 'n' rather than '0'.
                 lastCheckpoint.logOffset = -1;
-                const checkpointMessage = JSON.stringify(lastCheckpoint);
+                const checkpointMessage = `Restoring checkpoint from latest summary. Seq number: ${JSON.stringify(lastCheckpoint.sequenceNumber)}`;
```

Comment:
I think you don't need to call JSON.stringify here. `lastCheckpoint.sequenceNumber` would be a number and it will be automatically converted to string when added in the message. See example below on line 162.